.github/workflows/conda-build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
     steps:
 
     - name: Checkout code
-      uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+      uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
```

Comment:
I opened Quansight/open-gpu-server#71 to ask how to update the cirun images. In the meantime, this must be reverted.